blinkt.go

```diff
@@ -40,6 +47,25 @@ func writeByte(val int) {
 	}
 }
 
+func (bl *Blinkt) InitStopper() {
```

Comment:
I've renamed to `SetClearOnExit()` and added a boolean parameter in the process.  I thought about leaving the function parameter-less and only including the function where clear on exit is required. I decided that adding the parameter provides users with a choice and an ability to be explicit about whether it should clear on exit, improving readability within the examples.